io/jvm/src/main/scala/fs2/io/net/unixsocket/JdkUnixSockets.scala

```diff
@@ -42,27 +43,32 @@ private[unixsocket] class JdkUnixSocketsImpl[F[_]: Files](implicit F: Async[F])
     extends UnixSockets.AsyncUnixSockets[F] {
   protected def openChannel(address: UnixSocketAddress) =
     Resource
-      .make(F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX)))(ch =>
-        F.blocking(ch.close())
-      )
+      .make(
+        F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX)).evalOnVirtualThreadIfAvailable()
```

Comment:
Using `evalOnVirtualThreadIfAvailable` as a member function is more readable in my opinion. It doesn't get in the way on the actual code.

Reply:
Fair enough